modules/serverless-event-auth-applying-eventpolicy.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-event-authorization-eventpolicy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-event-auth-applying-eventpolicy_{context}"]
+= Applying an EventPolicy
+
+This section describes the end-to-end steps for applying an EventPolicy to secure event delivery. In the following reference example a Broker in `namespace-1` is configured to only accept events from a `PingSource` running in a different namespace `namespace-2`.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {ocp-product-title} cluster. 
+* You have enabled the `authentication-oidc` feature.
+
+.Procedure
+
+. Create two namespaces, deploy a Broker in `namespace-1`, and configure one `PingSource` in each namespace as per shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-2
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: broker
+  namespace: namespace-1
+---
+# PingSource in namespace-1
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: pingsource-1
+  namespace: namespace-1
+spec:
+  data: '{"message": "Hi from pingsource-1 from namespace-1"}'
+  schedule: '*/1 * * * *'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: broker
+      namespace: namespace-1
+---
+# PingSource in namespace-2
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: pingsource-2
+  namespace: namespace-2
+spec:
+  data: '{"message": "Hi from pingsource-2 from namespace-2"}'
+  schedule: '*/1 * * * *'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: broker
+      namespace: namespace-1
+----
+
+. Create an event-display service to show incoming events and add a Trigger to connect it to the Broker as per shown in the following example:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: namespace-1
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
+    spec:
+      containers:
+      - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
```

Comment:
maybe we can use a plain k8s service and deployment to not require serving at this point

Reply:
I have added a plain Kubernetes Deployment + Service version (no Knative Serving), with the Trigger still subscribing to the Service. Please check and let me know